src/ecalc/libraries/libecalc/common/libecalc/common/component_info/compressor.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class CompressorInputPressures(str, Enum):
```

Comment:
```suggestion
class CompressorInputPressures(str, Enum):
```

**Input**pressures is misleading. Not sure if we need/want enum here? Might be a good idea to have safe typing and avoid silly typos etc though..